apps/dashboard/src/components/primitives/control-input/control-input.tsx

```diff
@@ -117,21 +116,18 @@ export function ControlInput({
         value={value}
         onChange={onChange}
       />
-      <Popover open={!!selectedVariable} onOpenChange={handleOpenChange}>
```

Comment:
moved to reusable component that later is used in the Email editor when editing the variable